src/util.js

```diff
@@ -69,10 +69,8 @@ function replaceCIDbyTAG (dagNode) {
   return transform(dagNode)
 }
 
-exports = module.exports
```

Comment:
Does not  doing `exports` anymore break anything? I forgot all the details about `exports` vs. `module.exports`.

Reply:
The change is because the new linting rules forbid assigning `exports`